### PR TITLE
[CHAT-1159] import channel messages

### PR DESCRIPTION
**Pull request by thesyncim**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: channel.go [hunk 1]
```diff
@@ -104,6 +104,10 @@ func (q queryResponse) updateChannel(ch *Channel) {
 	}
 }
 
+type SendBulkMessagesResponse struct {
+	Messages []Message `json:messages`
```
Comment (gumuz) [inline channel.go:108]: I think it should be json:"messages", like the linter suggests

Comment (thesyncim) [reply channel.go:108]: ouch :D

**Review by gumuz — APPROVED** [review]

File: channel.go [hunk 2]
```diff
@@ -212,6 +216,27 @@ func (ch *Channel) RemoveMembers(userIDs []string, message *Message) error {
 	return nil
 }
 
+// ImportChannelMessages is a batch endpoint for inserting multiple messages.
+func (ch *Channel) ImportChannelMessages(messages ...*Message) (*ImportChannelMessagesResponse, error) {
```
Comment (ferhatelmas) [inline channel.go:220]: could you add it to channel interface ?

**Review by ferhatelmas — COMMENTED** [review]
see one comment otherwise good

File: channel.go (hunk 2 above)
Comment (thesyncim) [reply channel.go:220]: done

Comment (ferhatelmas) [reply channel.go:220]: thanks but we need to rename it to `ImportMessages`, it's already a method of channel

Comment (thesyncim) [reply channel.go:220]: good point, fixing it

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**